### PR TITLE
Bug 1162040 - updating accessibility for gaia-icons used in Usage app.

### PR DESCRIPTION
**Pull request by yzen**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bb28ea46caef25b81286095701c52dc11438a36f

File: apps/costcontrol/locales/costcontrol.en-US.properties [hunk 1]
```diff
@@ -102,6 +102,9 @@ long-date-format = %B %d, %Y
 sms-sent = SMS sent
 call-made = Calls made
 settings = Settings
+# LOCALIZATION NOTE (settingsButton): label for settings button. The following
+# string is spoken by screen readers and not shown on the screen.
+settingsButton.ariaLabel = Settings
```
Comment (gitmai) [inline apps/costcontrol/locales/costcontrol.en-US.properties]: Please, to maintain the same coding style,  don't camelize the tag settingsButton (use settings-button.ariaLabel)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7878134a10f0953fdcba83b46a20750cc5fea619

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7878134a10f0953fdcba83b46a20750cc5fea619

**Pull request merged**